src/fragments/sdk/push-notifications/ios/getting-started.mdx

```diff
@@ -187,7 +189,7 @@ Use the following steps to connect push notification backend services to your ap
         // See https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter for more details
         completionHandler()
     }
-}
+    }
```

Comment:
```suggestion
}
```

Reply:
Without the extra spacing, the code block does not display correctly. However, I think the functions inside `extension` need to be indented an extra level as well, I just made that change.